systemvm/patches/debian/config/opt/cloud/bin/cs/CsConfig.py

```diff
@@ -66,8 +66,13 @@ def is_router(self):
     def get_domain(self):
         return self.cl.get_domain()
 
+    def use_extdns(self):
+        return self.cmdline().idata().get('useextdns', 'false') == 'true'
+
     def get_dns(self):
+        conf = self.cmdline().idata()
         dns = []
+<<<<<<< e09480728f4ed77d0bd0bdc6a6c0966d90f7ee26
```

Comment:
I think you want to squash these out with the last commit in your PR. We wouldn't want this in our history you dutch commercial fan ;)